### PR TITLE
SVA-221 | Update API deployment instructions on how to safely deploy to AWS

### PR DESCRIPTION
**Pull request by joefhall**
- [Description](#description)
  - [Type of change](#type-of-change)
- [Checklist](#checklist)

# Description

- Updated deployment instructions -- currently this looks like the easiest, hopefully foolproof way to deploy API changes
- As part of testing this, if you have access to our AWS account, please can you try it yourself as a double-check that it works

Fixes SVA-221

## Type of change

Please delete options that are not relevant
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] This change requires a documentation update

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (if required)
- [] My changes generate no new warnings
- [] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: DEPLOYMENT.md [hunk 1]
```diff
@@ -48,13 +48,20 @@ For support, please @ David Calder in the [volunteer-app](https://scottishtechar
...
+5. Go into the `api` directory of your new copy of the repo, e.g. `cd volunteer-app-aws-deployment/api`
+6. In that `api` directory, zip it up into a file: `zip ../../myapp.zip -r * .[^.]*`  This `myapp.zip` file will be created in the directory above `volunteer-app-aws-deployment`
+7. Go to the AWS Management Console and navigate to [Elastic Beanstalk](https://eu-west-2.console.aws.amazon.com/elasticbeanstalk/home?region=eu-west-2#/environments).
+8. In [Application versions](https://eu-west-2.console.aws.amazon.com/elasticbeanstalk/home?region=eu-west-2#/application/versions?applicationName=volunteer-app), Upload the myapp.zip that you created in step 6.
+9. Now select the version label you've just created and then select Action > Deploy
+10. Go to the environment dashboard and check the version label has updated and the Health is OK. If not, check the Logs (menu on the left hand side) or ask someone else in the team for help.
+11. Delete the new directory containing copy of the repo you made in step 2 and everything in it, e.g. `rm -rf volunteer-app-aws-deployment/`
```
Comment (jppferguson) [inline DEPLOYMENT.md:64]: Wondering if we should add another step in between 10-11 to do a quick check that data is being pulled through on at least one of the endpoints. eg. `http://18.134.220.155/projects`?

**Review by jppferguson — COMMENTED** [review]
Just one suggestion, other than that it makes sense to me. I'm not able to run through the process though as I was never able to get access to AWS 😭

**Review by CatMac228 — APPROVED** [review]
Approved :)

File: DEPLOYMENT.md (hunk 1 above)
Comment (joefhall) [reply DEPLOYMENT.md:64]: Great shout @jppferguson -- will do! 👍

Comment (joefhall) [reply DEPLOYMENT.md:64]: Sorry to hear you've not got access to the AWS account -- maybe message David Calder on Slack and he can help?

**Review by DougieWougie — APPROVED** [review]
Nice one @joefhall !

**Comment by joefhall** [comment]
Thanks @CatMac228 @jppferguson @DougieWougie !

**Pull request merged**